packages/gamut-labs/src/LayoutMenu/LayoutMenu.tsx

```diff
@@ -25,6 +25,10 @@ export type LayoutMenuProps = {
    * Breakpoint above which the menu button displays as a full sidebar
    */
   breakpoint?: 'xs' | 'sm' | 'md' | 'lg' | 'xl';
+  /**
+   * An additional link section on top of the layout menu
+   */
+  topLinkSection?: React.ReactNode;
```

Comment:
Allowing any `React.ReactNode` means any sort of button/text/whatnot could be added in. IMO we'd generally want to be a bit more restrictive in the designs and only allow the specific kinds of things this area is intended to include: in this case, button links. 

How about making this a `Section` or `SectionItem` the way the other sections are set up?

```suggestion
  topLinkSection?: SectionItem[];
```

Reply:
I see your point. I was following the way children act here, since we'd want to conditionally render some top links based on if you belong to an organization + your organization has its own content and making a component do that and pass it into this just seemed so nice.

If your organization has content, we want that link on top of all the links and the full catalog link below it. If not, we just want just the full catalog link on top. I'll edit this to make it a SectionItem[], and just do some calculations in portal after we fetch everything.